### ISSUE TITLE
Max limit when generating list & can't have multiple folders/zip files or they overwrite each other

### ISSUE THREAD

**Issue by GoldenTankCo** [TeamTernate/infinite-music-discs#88]
This problem is rather annoying but maybe there is a solve I haven't tried but I have a large playlist (1403 songs) and putting them in and trying to generate it says the pack.png file is missing but I can clearly see it and it won't let me continue. I'm assuming past a certain point it just stops trying so idk.

This leads into the 2nd problem so I tried breaking it up by letter and putting them in separate folders/zip files and putting in the first folder by itself runs smoothly with no hiccups. But introducing a 2nd folder and beyond that the 2nd folder overwrites the songs. For example the 4 AC/DC songs I have for letter A work fine but next on the alphabet I have C for C418, Celeste, Cuphead etc. As for the first song for C418 I have is Alpha so playing Alpha works fine but going back to play the first song on the list for A being Dirty Deeds Done Dirt Cheap, it plays Alpha with the noise from Disc 11 mixed in. 

Some of the solutions I've tried like generating 1 folder/file (like I said it doesn't work), I tried manually splicing the files together but that is also a no, finally I realized it could be human error so I let the computer take groups of the songs and do it so I don't do something wrong. As of writing this I'm testing the 3rd method so I can add an update when I try it, but if this could get fixed that would really helpful.

Please and Thank You

**Comment by bradytheinventor** [comment]
What happens if you generate a pack without providing a pack.png file, is it able to process all 1400 tracks? What happens if you generate a pack with fewer tracks, does it still give an error? Did you move the pack.png file somewhere else after you added it to the pack but before you clicked the generate button? This error message pops up if you point the app to a pack.png file but it can't find a valid file at the path you gave it.

For the 2nd problem, unfortunately you can't mix and match packs currently. Packs use the same CustomModelData space (first disc is always CustomModelData:0, second is always CustomModelData:1, etc.) so if you include multiple packs they will overlap with each other and interfere. I would like to make it so you can add several different Infinite Music Discs packs at the same time and have them work together, but right now that's not supported.

**Comment by GoldenTankCo** [comment]
OK so for the first suggestion I'll give that a try and report back. By fewer tracks I don't know the limit before the pack.png starts acting funny. I tried stringing about 500-ish tracks and I think that's where it starts to break. I kept the pack.png in the same spot it was when I downloaded the desktop app from here. The path is the same as when it opens but I'll tinker with it to see what I can get done. 

I understand about the 2nd issue now with your explanation, While its a tad annoying I found a workaround. By having the resource packs all in the folder its fine but when I'm on the server with my friend I figured out if I want to play a song in that specific letter group I can just swap the correct data pack in and it works after reloading.

If you ever can get it working I would be incredibly thankful as the work you and the team have done cannot be understated.
